src/sonic-yang-models/yang-models/sonic-extension.yang

```diff
@@ -7,10 +7,18 @@ module sonic-extension {
 
     description "Extension yang Module for SONiC OS";
 
+    revision 2021-02-23 {
+        description "Adding extensions as per Sonic Yang Guidelines";
+    }
+
     revision 2019-07-01 {
         description "First Revision";
```

Comment:
'First revision' date seems not right, please update.

Reply:
All the yang files have this same date as "First Revision". I didn't see anything wrong here. Latest revision should always be placed on top, else compilation gives warning.